packages/cxl-ui/scss/cxl-app-layout/_layout.scss

```diff
@@ -23,6 +23,15 @@ $toggle-icon: "lumo:angle-right";
 :host([layout^="2c"]) {
   main {
     > slot:not([name]) {
+      /**
+      * Force container height.
+      *
+      * border-box is used to avoid overflow caused by padding.
+      * 60px is action bar height
+      */
+      box-sizing: border-box;
```

Comment:
Now needing `border-box` signals this is probably not the right foundational solution, but lack of bandwidth means it'll just have to be improved later.